### PR TITLE
Event Card Group and Sidebar (Closes #548 & #549)

### LINKED ISSUE #548: Create Event Card Group view of Events
Match Quickstart 1 exactly

### PR DESCRIPTION
**Pull request by smayden**
Initial work on views and styling for Event Card Group and Sidebar Event Block

## Description
This is ready!

Created Grid View (block) on the Events View:

image

And the Grid Sidebar:

image

Some questions:
-In QS 1 both of the blocks use the card display - is that desirable? If summaries are entered it takes up a lot of space when displayed on the node (often longer than the actual event content in the main content region)
-I haven't been able to get Event Card Group to actually create cards of equal height (despite trying multiple things). Does this matter? (we currently don't have the grid views with cards for person or news having equal height, but it would be nice eventually)
-You can see from the screenshot that things look a bit strange for repeating dates. What is the desired behavior here? (calendar view also shows multiple dates right now)
-Feedback on design/styling/spacing appreciated.

Probo build: https://7071b8d9-e3eb-408a-80b5-95add93e50eb.probo.build
Grid view: Probo build: https://7071b8d9-e3eb-408a-80b5-95add93e50eb.probo.build/test-page-event-grid
Sidebar: https://7071b8d9-e3eb-408a-80b5-95add93e50eb.probo.build/events/arizona-womens-basketball-vs-texas-am

## How Has This Been Tested?
tested locally and in Probo

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I've added this Pull Request to the AZ Quickstart Github project and set it to "Review in progress".
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by smayden** [comment]
Some notes from Slack feedback:
-Restore view settings for full calendar (or figure out how /calendar got messed up)
-Update padding
-Make title smaller
-Make date smaller

**Comment by smayden** [comment]
Made a few different updates:
- Design tweaks (spacing, text/heading sizes)
- Restored the full calendar view (which inadvertently got broken) and renamed it to full calendar for clarity
- Added header on view for sidebar
- Added contextual filter on sidebar to exclude displaying same event as node

File: modules/custom/az_event/config/install/core.entity_view_display.node.az_event.az_card.yml [hunk 1]
```diff
@@ -0,0 +1,308 @@
...
+      format_settings:
+        element: h2
+        show_label: false
+        label_element: h3
+        label_element_classes: ''
+        attributes: ''
+        effect: none
+        speed: fast
+        id: ''
+        classes: 'card-title text-red h5 mt-0'
```
Comment (danahertzberg) [inline modules/custom/az_event/config/install/core.entity_view_display.node.az_event.az_card.yml:236]:
```suggestion
        classes: 'card-title text-red h5 mt-0 mb-2'
```

File: modules/custom/az_event/config/install/core.entity_view_display.node.az_event.az_card.yml [hunk 2]
```diff
@@ -0,0 +1,308 @@
...
+      format_settings:
+        element: div
+        show_label: false
+        label_element: h3
+        label_element_classes: ''
+        attributes: ''
+        effect: none
+        speed: fast
+        id: ''
+        classes: 'text-muted font-weight-bold small mt-0 mb-0'
```
Comment (danahertzberg) [inline modules/custom/az_event/config/install/core.entity_view_display.node.az_event.az_card.yml:218]:
```suggestion
        classes: 'text-muted font-weight-bold small'
```
(margin classes aren't needed here)

**Review by danahertzberg — COMMENTED** [review]
I added some style recommendations to your code. I would also recommend ditching the AP styles in this case so we can show only the next upcoming date with less words (9 a.m. on Tuesday instead of 9 a.m.-5 p.m. etc etc)

**Comment by smayden** [comment]
Committed style suggestions from @danahertzberg 

I think the question of the date/time display will need further discussion. Removing the end time is not related to AP Style here (although that would affect if we wanted to abbreviate the months), but I think rather has to do with how the When field is set up (and is used across all the Events views).

**Review by danahertzberg — APPROVED** [review]

**Review by tadean — APPROVED** [review]

**Pull request merged**